plugins/package-managers/node/src/main/kotlin/npm/Npm.kt

```diff
@@ -179,13 +187,41 @@ class Npm(override val descriptor: PluginDescriptor = NpmFactory.descriptor, pri
 
         return process.extractNpmIssues()
     }
+
+    private fun requestAllPackageDetails(projectModuleInfo: ModuleInfo) {
+        runBlocking {
+            withContext(Dispatchers.IO.limitedParallelism(20)) {
```

Comment:
Why this special value ? Is it from empirical testing ?

Reply:
It aligns with other similar code that was introduced by @mnonnenmacher in 23c9bb0d018f92a4444167293e677ae8c5f6f561. But I agree that we should afterwards probably extract a constant.